### PR TITLE
Issue #SB-21876 fix: API documentation for portal backend - form API

### PR DESCRIPTION
**Pull request by gouravmore**
Issue #SB-21876 fix: API documentation for portal backend - form API

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @gouravmore,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://639.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://639.qa.docs.sunbird.org/api-documentation/apis/form// <br />  <br/> **Modified sections are -** <br />

File: apis/form/formapi.yaml [hunk 1]
```diff
@@ -1,40 +1,48 @@
...
+    - The Form APIs allow you to create and update metadata form fields based on
+    type, subtype and action. Use the APIs to configure form fields as required
+    for the specific form.
+
     - The URL for Form API is `/data/v1/form/`.
-  version: "1.0"
+  version: '1.0'
 servers:
-- url: http://staging.open-sunbird.org/api/data/v1/form
+  - url: 'http://staging.open-sunbird.org/api/data/v1/form'
```
Comment (SMYALTAMASH) [inline apis/form/formapi.yaml:12]: Please add the url as `https://staging.open-sunbird.org/api` only and add the rest of the part in the api path below.

Comment (gouravmore) [reply apis/form/formapi.yaml]: ok

**Comment by keshavprasadms** [comment]
Hello @gouravmore,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://639.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://639.qa.docs.sunbird.org/api-documentation/apis/form// <br />  <br/> **Modified sections are -** <br />

File: apis/form/formapi.yaml [hunk 2]
```diff
@@ -1,40 +1,48 @@
...
+    for the specific form.
+
     - The URL for Form API is `/data/v1/form/`.
-  version: "1.0"
+  version: '1.0'
 servers:
-- url: http://staging.open-sunbird.org/api/data/v1/form
+  - url: 'http://staging.open-sunbird.org/api'
 paths:
   /create:
```
Comment (SMYALTAMASH) [inline apis/form/formapi.yaml:14]: this should be /data/v1/form/create @gouravmore

**Pull request merged**